cvat/apps/dataset_manager/task.py

```diff
@@ -735,41 +718,36 @@ def _init_tracks_from_db(self):
                     "shape__id",
                     "shape__frame",
                     "shape__outside",
-                    "shape__attribute__spec_id",
-                    "shape__attribute__value",
-                    "shape__attribute__id",
                 ],
             },
             field_id="id",
         )
 
+        labeledtrack_attributes = _receive_attributes_from_db(
+            self.db_job.labeledtrackattributeval_set,
+            "track_id",
+        )
+        trackedshape_attributes = _receive_attributes_from_db(
+            self.db_job.trackedshapeattributeval_set,
+            "shape_id",
+        )
+
         tracks = {}
         elements = {}
         for db_track in db_tracks:
             if not db_track["shapes"]:
                 continue
-            db_track["shapes"] = merge_table_rows(
-                db_track["shapes"],
-                {
-                    "attributes": [
-                        "attribute__value",
-                        "attribute__spec_id",
-                        "attribute__id",
-                    ]
-                },
-                "id",
-            )
 
             # A result table can consist many equal rows for track/shape attributes
             # We need filter unique attributes manually
-            db_track["attributes"] = list(set(db_track["attributes"]))
+            db_track["attributes"] = list(set(labeledtrack_attributes[db_track["id"]]))
```

Comment:
Do you know why there can be duplicates?

Reply:
Not really. Technically easy as this is one to many relation, however on practice IDK
I wanted to ask @SpecLad about it

And why we do not check the same for shapes and tags.

Reply:
I guess  the answer is that there are no DB constraints to prevent duplicated spec_ids per shape_id. Ok, let's keep this as it is.

Reply:
If this is the case, maybe we need to add some deduplication means on saving or use attrs with max `id` for each `spec_id`.